### PR TITLE
Add duplicate debug aggregatable reports

### PR DESCRIPTION
**Pull request by alexmturner**
Resolves issue #375.

### CODE REVIEW

**Comment by alexmturner** [comment]
@linnan-github could you PTAL? Thanks!

File: AGGREGATE.md [hunk 1]
```diff
@@ -315,6 +308,25 @@ future versions of the API), the endpoint URL should also change.
 **Note:** The browser may need some mechanism to ensure that the same set of
 keys are delivered to different users.
 
+#### Optional: extended debugging reports
+
+If [debugging](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-extended-debugging-reports)
+is enabled, additional debug fields will be present in aggregatable reports.
+The `source_debug_key` and `trigger_debug_key` fields match those in the
+event-level reports. The `debug_cleartext_payload` field contains the
```
Comment (linnan-github) [inline AGGREGATE.md:316]: Shall we mention that `debug_cleartext_payload` and `debug_mode` will only be present if both source and trigger debug keys are present?

Comment (alexmturner) [reply AGGREGATE.md]: Thanks, updated!

File: AGGREGATE.md [hunk 2]
```diff
@@ -315,6 +308,26 @@ future versions of the API), the endpoint URL should also change.
...
+
+If [debugging](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-extended-debugging-reports)
+is enabled, additional debug fields will be present in aggregatable reports.
+The `source_debug_key` and `trigger_debug_key` fields match those in the
+event-level reports. If both the source and trigger debug keys are set, there
+will be a `debug_cleartext_payload` field included in the report. It will
+contain the base64-encoded cleartext of the encrypted payload to allow downstream
+systems to verify that reports are constructed correctly. If both debug keys are
+set, the `shared_info` will also include the flag `"debug_mode": "enabled"` to
+allow the aggregation service to support debugging functionality on debug reports.
```
[the remark this answers is not in the document]
Comment (alexmturner) [reply AGGREGATE.md]: Ah good catch, this is definitely confusing. Went with "these reports" instead.

**Review by linnan-github — APPROVED** [review]

**Review by johnivdel — APPROVED** [review]

**Pull request merged**